src/pages/donate/form.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Button, Form, Col, InputGroup, } from 'react-bootstrap';
+
+const MONTHS_ARR = Array.from({length: 12}, (_, i) => i + 1);
+const CURR_YEAR = new Date().getFullYear();
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from
+const range = (start, stop, step) => Array.from({ length: (stop - start) / step + 1}, (_, i) => start + (i * step));
```

Comment:
Can you move this an `src/util`. Seems like a function that would be useful throughout the app.